psasim/Makefile

```diff
@@ -0,0 +1,64 @@
+CFLAGS ?=  -Wall -std=c99
+INCLUDE := -I./include/
+DESTDIR ?= /usr/local
+PREFIX := libpsaff
+BUILDDIR ?= bin
```

Comment:
If I'm not wrong `BUILDDIR` seems not to be used anywhere (a part from the `clean` target, but that's not relevant)